### ISSUE TITLE
Version 0.1.3 throws error: Uncaught ReferenceError: branchName is not defined

### ISSUE THREAD

**Issue by avaly** [dbslone/git-tabs#5]
1. Load Atom with a git repository

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 16.04.2
**Thrown From**: [git-tabs](https://github.com/dbslone/git-tabs) package 0.1.3

### Stack Trace

Uncaught ReferenceError: branchName is not defined

```
At /home/user/.atom/packages/git-tabs/lib/tab-manager.js:54

ReferenceError: branchName is not defined
    at TabManager.onDidChangeStatuses (/packages/git-tabs/lib/tab-manager.js:54:38)
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app/node_modules/event-kit/lib/emitter.js:129:1)
    at Task.callback (/usr/share/atom/resources/app/src/git-repository.js:412:1)
    at /usr/share/atom/resources/app/src/task.js:67:1
    at /usr/share/atom/resources/app/src/task.js:128:1
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app/node_modules/event-kit/lib/emitter.js:129:1)
    at ChildProcess.<anonymous> (/usr/share/atom/resources/app/src/task.js:80:1)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at process.nextTick (internal/child_process.js:744:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

### Commands

```
     -1:32.2.0 multirow-tabs:toggle (atom-workspace.workspace.scrollbars-visible-always)
     -1:32.2.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -1:26 core:close (div.panels-item)
```

### Non-Core Packages

```
atom-beautify 0.29.26 
atom-npm-outdated 0.11.5 
open-project-file 1.1.1 
autocomplete-modules 1.6.10 
busy-signal 1.4.1 
dash 1.7.1 
docblockr 0.10.3 
editorconfig 2.2.2 
file-icons 2.1.6 
gist-it 0.9.2 
git-tabs 0.1.3 
git-time-machine 1.5.9 
highlight-selected 0.13.1 
hyperclick 0.0.40 
hyperlink-hyperclick 1.3.4 
intentions 1.1.2 
js-hyperclick 1.11.0 
language-babel 2.65.0 
language-docker 1.1.8 
language-terraform 0.8.1 
linter 2.1.4 
linter-eslint 8.2.1 
linter-jsonlint 1.3.0 
linter-stylelint 3.10.0 
linter-ui-default 1.6.0 
markdown-writer 2.6.5 
minimap 4.28.2 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
node-debugger 1.10.1 
open-git-modified-files 0.2.4 
open-recent 5.0.0 
pigments 0.39.1 
prettier-atom 0.31.1 
pretty-json 1.6.3 
project-manager 3.3.5 
rainbow-tabs 2.2.1 
require-opener 2.0.3 
screen-recorder 1.2.3 
todo-show 1.11.0 
```

**Comment by avaly** [comment]
Most likely introduced in https://github.com/dbslone/git-tabs/commit/9e55f3642145c36ea28e597fd9e960fbe211cfb7

**Comment by dbslone** [comment]
@avaly the error should be resolved in v0.1.4. Let me know if the problem persists
